service/committer/committer/committer.go

```diff
@@ -65,6 +68,9 @@ func (c *Committer) Run() {
 	}
 
 	for {
+		if !c.running {
+			break
```

Comment:
do we need to add a channel to notify the shutdown signal, it would be more precise.

Reply:
Because it is a single process now, so i did not use `channel` or `atomic value`  - a simple boolean is easy to use and understand. 
Let me also check others' opinions. @KeefeL